server/src/main/java/org/gluu/oxtrust/action/uma/UpdateResourceAction.java

```diff
@@ -92,7 +92,7 @@
 	private boolean update;
 	private String scopeSelection="Scopes";
 	
-	private List<OxAuthClient> clientList;
+	private List<OxAuthClient> clientList = new ArrayList<OxAuthClient>();;
```

Comment:
Why we init this variable here? I think we should do this in action method which jsf call

Reply:
clientList  is used in lots of methods like add, remove, update client thats why i initialize it to prevent null pointer method,

Reply:
I added this commit to this PR: https://github.com/GluuFederation/oxTrust/pull/999/commits/ea96913470f30ddbc336a04492e563cf06f7eb1b

Is it work as expected?